package.json

```diff
@@ -25,13 +25,14 @@
     "build:v3": "pnpm run dist:select --force-version=3",
     "build": "vite build",
     "build:all": "pnpm build:v2 && pnpm build:v3",
-    "build:storybook": "storybook build",
+    "build:storybook": "STORYBOOK=true storybook build",
     "build:tailwind": "tailwindcss -o dist/style.css --minify",
     "build:types": "vue-tsc -p tsconfig.build.json",
+    "generate-icons": "node scripts/generate-icons.js",
     "resolve:alias": "tsc-alias -p tsconfig.build.json",
     "dev": "vite",
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix",
+    "lint": "eslint --cache .",
+    "lint:fix": "eslint --cache . --fix",
```

Comment:
I removed the `--cache` option because we don't actually need it here.
If you noticed slow linting this is because we don't properly ignore some dist folders. can you replace it with `--ignore-path .gitignore` and check?

Reply:
Yeah it fix it, I will add the .eslintignore instead.